content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,151 @@
-
-Mentions in the chat and on the chatter
----------------------------------------
-
-To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
-`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
-on their communication settings.
-
-.. image:: discuss/chat-windows.png
-   :alt: View of a couple of chat window messages for Odoo Discuss.
-
-.. tip::
-   When a user is mentioned, the search list (list of names) suggests values first based on the
-   task's followers, and secondly on employees. If the record being searched does not match with
-   either a follower or employee, the scope of the search becomes all partners.
+At the top of the :guilabel:`Direct Message` page, click the name of the direct message to change
+the group name.
 
 User status
 -----------
 
 It is helpful to see what colleagues are up to and how quickly they can respond to messages by
 checking their *status*. The status is shown on the left side of a contact's name on the
-:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
+:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *Chatter*.
 
 - Green = online
 - Orange = away
 - White = offline
 - Airplane = out of the office
 
-.. image:: discuss/status.png
-   :height: 300
-   :alt: View of the contacts' status for Odoo Discuss.
+Deleting a direct message conversation
+--------------------------------------
+
+To delete direct message conversations from the :guilabel:`Direct Message` view, click the
+:guilabel:`X` next to the name of the direct message conversation. This does not delete the
+:guilabel:`Direct Messages` in the direct message conversation. The direct message conversation's
+history is visible when another direct message with the same person or group is created.
+
+.. _discuss/meetings:
+
+Meetings
+========
+
+In *Discuss*, :guilabel:`Meetings` are video calls. To start a meeting from the *Discuss*
+application home page, click :guilabel:`Start a meeting` in the top left corner. To start a meeting
+from a direct message, click the :icon:`fa-phone` :guilabel:`Start a Call` icon in the top right.
+
+.. image:: discuss/meeting.png
+   :alt: View of a Meeting in Odoo Discuss.
+
+Once a meeting has been started, the buttons on the bottom of the call screen can be used to
+:icon:`fa-microphone` :guilabel:`Mute/Unmute`, :icon:`fa-headphones` :guilabel:`Deafen/Undeafen`,
+and :icon:`fa-video-camera` :guilabel:`Turn camera on/off`. Clicking the :icon:`fa-ellipsis-v`
+:guilabel:`More` icon allows the user to :guilabel:`Raise Hand`, :guilabel:`Share Screen`, or
+:guilabel:`Enter Full Screen`.
+
+Configuration
+=============
+
+Selecting :guilabel:`Configuration` at the top of the Discuss homepage triggers a dropdown menu of
+options including :guilabel:`Notification` :guilabel:`Voice & Video`, and
+:guilabel:`Canned Responses`.
```

Comment:
I'm on the Discuss homepage, and I'm actually not seeing any 'Configuration' option, or any other option. Can you please double-check this to ensure the accuracy of the statement. Do certain access rights need to be enabled for this? Some clarification would be helpful, thanks! 👍

Reply:
![Screenshot 2024-08-20 122318](https://github.com/user-attachments/assets/7fa0b061-c082-420f-884e-331b9217a429)